### ISSUE TITLE
Rune text Symbols issue

### ISSUE THREAD

**Issue by CypressBz** [hrzntal/horizon#1008]
### Client Version

514.whatever latest is

### Description of the Issue

![image](https://user-images.githubusercontent.com/7775562/170811350-ebf4e0b5-7a62-4cff-8a99-ce1524d93dd7.png)
Got this instead of an apostrophe. It happens in admin .d deadchat too, though, unrelated.

### Reproduction Steps

use apastrophes to denote possession or plurarity.

**Comment by Azarak** [comment]
Possible input sanitization misshap

**Comment by atakiya** [comment]
> Possible input sanitization misshap

Exactly. Either its double sanitized or doesn't need to be sanitized.
